application/models/TemplateConfig.php

```diff
@@ -1054,6 +1054,24 @@ public static function formatToJsonArray($oFiled, $bConvertEmptyToString = false
         return $jFiled;
     }
 
+    /**
+     * Convert the theme options to json.
+     * @param array|object $options the filed to convert
+     * @return string  json
+     */
+    public static function convertOptionsToJson($options)
+    {
+        $optionsArray = [];
+        foreach ($options as $option => $optionValue) {
+            // Clean every value from newlines, tabs and blank spaces for options
```

Comment:
It potentially broken some theme…

https://gitlab.com/SondagesPro/SurveyThemes/skelvanilla/-/blob/master/views/options/options_logocolors.twig?ref_type=heads#L301

Can be reviewed on another solution ?

Reply:
This was already happening on formatToJsonArray.

Can I please ask you to expand on what is being broken?

Reply:
i write _potentially_ the textearea result are used for css style here. Need line feed.

Here : it's when export./import lss ? Must try with current version and see what happen, report the issue if issue exist.

Reply:
I am not sure why lin feeds were removed. We are just porting the code.

Still, why do you need line feed for CSS?

Reply:
> Still, why do you need line feed for CSS?

It's for simple user who, just to add some css to specific survey. You realy want to remove line feed for simple user after copy survey ?

And here : it's a sample, i really think we must not update data … if data have line feed : data export as lss and imported aftsr must have line feed.

Reply:
> This was already happening on formatToJsonArray.
> Can I please ask you to expand on what is being broken?

Can not test currently with skelvanila on 6
But on 5.X : linefeed respected when export/import lss

I try to create a quick compatible version  …

Reply:
> You realy want to remove line feed for simple user after copy survey ?

I don't want to remove anything.
I think this is a side effect. The code which was there maybe was not working and now it is.